packages/contentful--create-contentful-app/src/app-actions/javascript/build-actions.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs');
+const { readdirSync, readFileSync, writeFileSync, mkdirSync } = fs;
+
+const buildActions = () => {
+  try {
+    console.log('Building app actions');
+    mkdirSync('build/actions', { recursive: true });
+
+    const actionsFiles = readdirSync('actions', {
+      encoding: 'utf-8',
+    });
+
+    actionsFiles.forEach((fileName) => {
+      const action = readFileSync(`actions/${fileName}`, {
+        encoding: 'utf-8',
+      }).toString();
+
+      writeFileSync(`build/actions/${fileName}`, action);
+      console.log('App actions successfully built');
+    });
+  } catch (e) {
+    console.log(e);
```

Comment:
I think we should halt the whole process here, as this is critical failure.

Reply:
Yeah I think you're right

Reply:
No, I meant that it should end process with an error. Something like this:
https://github.com/contentful/create-contentful-app/blob/862b997f25b617691a8a411abe27377adc86a3ee/packages/contentful--app-scripts/lib/utils.js#L92

Reply:
@kado wdyt?